contrib/initramfs/hooks/Makefile.am

```diff
@@ -1,10 +1,12 @@
 hooksdir = /usr/share/initramfs-tools/hooks
 
 hooks_SCRIPTS = \
-	zfs
+	zfs \
+	zfsunlock
 
 EXTRA_DIST = \
-	$(top_srcdir)/contrib/initramfs/hooks/zfs.in
+	$(top_srcdir)/contrib/initramfs/hooks/zfs.in \
+	$(top_srcdir)/contrib/initramfs/hooks/zfsunlock.in
```

Comment:
I don't see any substitutions in `zfsunlock.in`. Can't this just be `zfsunlock` in the source tree? Then you shouldn't need it in `EXTRA_DIST`. You will need to change `$(hooks_SCRIPTS):%:%.in` to something like `$(hooks_SCRIPTS):zfs.in`.